ui/.yarn/releases/yarn-1.22.19.js

```diff
@@ -71633,9 +71633,9 @@ declare var __webpack_require__: mixed;
 
```

Comment:
Not really sure how this got in here - need to remove - happened after merging main 🤷